### ISSUE TITLE
Setting remote ID's

### ISSUE THREAD

**Issue by RemBrandNL** [CoMPaTech/esphome_ct#8]
I set everything up and it show up in ESPHome. I set the ventilation to low/mid/high so that all seems to work. Can't figure out two things though. How can I set the RemoteID1/RemoteID2 and how can I find them? I set the logger to VERY_VERBOSE, but nowhere the ID's seem to popup. Is this an issue with code or with me :-)

Cheers,
Remco

Setup is a wmos d1 mini and cr1101 using the yaml provided here.

**Comment by CoMPaTech** [comment]
Moved you issue from the old repository to this new one. As custom components are deprecated (but still work) - and some users had issues, I decided to start on making a more proper external component. The current code is 'working' but still less than optimal. 

Also see #7 which is closely tied to this - but I have made no effort to (yet) to attempt at re-reading the RemoteIDs in this approach.

**Comment by CoMPaTech** [comment]
@RemBrandNL @balk77 this is what it provides currently:

[17:36:06][D][c1101_fan:234]: Debug - RemoteID1: 
[17:36:06][D][c1101_fan:235]: Debug - RemoteID2: \xc5@̓I\xxx
[17:36:06][D][c1101_fan:236]: Debug - LastID: c5-40-e-cd-83-49-xx-xx
[17:36:14][D][c1101_fan:234]: Debug - RemoteID1: 
[17:36:14][D][c1101_fan:235]: Debug - RemoteID2: jY\x96\xa9\xa6\xxx\xxxf
[17:36:14][D][c1101_fan:236]: Debug - LastID: 6a-59-96-a9-a6-a9-xx-xx
[17:36:15][D][c1101_fan:234]: Debug - RemoteID1: 
[17:36:15][D][c1101_fan:235]: Debug - RemoteID2: jY\xb6\xa9\xa6\xxx\xxxf
[17:36:15][D][c1101_fan:236]: Debug - LastID: 6a-59-b6-a9-a6-a9-xx-xx
[17:36:16][D][c1101_fan:234]: Debug - RemoteID1: 
[17:36:16][D][c1101_fan:235]: Debug - RemoteID2: jY\x96\xa9\xa6\xxx\xxxf
[17:36:16][D][c1101_fan:236]: Debug - LastID: 6a-59-96-a9-a6-a9-xx-xx
[17:36:19][D][c1101_fan:234]: Debug - RemoteID1: 
[17:36:19][D][c1101_fan:235]: Debug - RemoteID2: W\xc3\xcaŏ\xdd\xxx\xxx
[17:36:19][D][c1101_fan:236]: Debug - LastID: 57-c3-ca-c5-8f-dd-xx-xx

**Comment by RemBrandNL** [comment]
Thanks @CoMPaTech! For now I went a different route and all works as I want. Since it does work to set the fan speed using the ESP, the only issue I had was knowing what the fan-speed is. I plugged my ITHO box into a smart socket with power metering, so based on the power draw I set the speed value accordingly. This combined with a humidity sensor in the bathroom (as long as it's higher than the 24h average of another room >> wet >> fan: high) and a temp sensor in the kitchen (temp higher than living room >> cooking >> fan: high) it all works like a charm and haven't used the remotes since. Should I run out of other projects, I might make another one using this repo. Thanks for your great work so far!
Remco
